app/views/login_activities/index.html.haml

```diff
@@ -8,14 +8,14 @@
       %tr
         = render 'layouts/sort_headers', search: @login_activity_search
 
-      - @login_activities.each do |login_activity|
-        %tr{ class: login_activity.success ? '' : 'alert-danger' }
-          %td= format_datetime(login_activity.created_at)
-          %td= login_activity.identity
-          %td= format_boolean(login_activity.success)
-          %td= login_activity.failure_reason
-          %td= login_activity.ip
-          %td= login_activity.user_agent
-          %td= login_activity.referrer
+      - @login_activities.each do |record|
```

Comment:
i've wanted to do this for a while, to change the model name to `record` so that the boilerplate is more resilient - we don't have to do this on every index page though but just did these while I was in there